tests/integration/docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
     image: minio/minio:latest
     command: server /data
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      test: [ "CMD", "bash", "-c", ":> /dev/tcp/127.0.0.1/9000" ]
```

Comment:
that's a new one to me? would it maybe be better to lock the tag to the previous release which supported curl?

Reply:
Im not sure that we have to lock it. of course we can but why? they started using new `base` images which now have no `curl`. in any case we will need to jump to the fresh `image` so why we can't do that right now?

Reply:
well if we used a release tag, this kind of thing wouldn't happen in the future, too -- and I think `curl` is more clear than using `/dev` filesystem (but that's me)

Reply:
that's true, it wouldn't happened but I think we want to use `latest` images(all our images have :latest prefix) so I think we have to follow this approach. not a big deal really. such cases happen too rarely.

Reply:
I quite liked the `curl` too, but this is the new recommended healthcheck for the official minio image, so let's go for it
with regards to locking the tag, I would like us to always use the latest available 3rd-party dependencies, and we immediately detected this issue thanks to the [daily scheduled CI run](https://github.com/G-Research/fasttrackml/actions/workflows/ci.yml?query=event%3Aschedule), which is the exact reason why we have it in the first place